### ISSUE TITLE
Test rvd_microservice in a local environment

### ISSUE THREAD

**Issue by otsakir** [RestComm/visual-designer#216]
Install both restcomm and RVD as docker instances and use nginx as a proxy to switch http request based on URL.

**Comment by otsakir** [comment]
I set up the following environment:

* On a separate docker network - 172.18.0.0
`docker network create --driver bridge --subnet=172.18.0.0/16 --gateway=172.18.0.1 rvdms_bridge`

* Created a docker nginx instance that worked a proxy
`docker run --name mynginx --network=rvdms_bridge --ip=172.18.0.2  -v ~/telestax/src/rvd-docker/nginx/html:/usr/share/nginx/html:ro -v ~/telestax/src/rvd-docker/nginx/conf:/etc/nginx:ro -p 80:80  nginx`

* Created a docker restcomm instance using community image
`docker run  -i -d --name=myrestcomm --network=rvdms_bridge --ip=172.18.0.3 -v /var/log/restcomm/:/var/log/restcomm/ -e RCBCONF_STATIC_ADDRESS="172.18.0.3" -e ENVCONFURL="https://raw.githubusercontent.com/RestComm/Restcomm-Docker/master/env_files/restcomm_env_locally.sh"  -p 9990:9990 -p 5060:5060 -p 5061:5061 -p 5062:5062 -p 5063:5063 -p 5060:5060/udp -p 65000-65050:65000-65050/udp restcomm/restcomm:latest`

* Created a docker rvd instance using latest local build from master
`docker run -it --network=rvdms_bridge --ip=172.18.0.4 -e DEBUG=true -e RESTCOMM_URL=http://172.18.0.2 --name=myrvd otsakir/rvd-docker:latest`

* Nginx was configured as follows:
```
events {
    worker_connections   2000;
}

http {

        server {
                location /restcomm-rvd/ {   
                        proxy_pass http://172.18.0.4/restcomm-rvd/;
                }

                location / {
                        proxy_pass http://172.18.0.3:8080/;
                }
        }
}
```
So that it would redirect requests to different instances based on URL. 

* No load balancing was done for SIP. 

* No https could be configured.

#### Conclusions
1. Both dashboard/RVD UI worked fine.
2. Calling 1234 succeeded
3. Calling RVD applications failed. The reason was that restcomm was looking for RVD at 172.18.0.3 (its own address) since that was the hostname of the machine. 
4. RVD resolved restcomm using the docker RESTCOMM_URL parameter pointing to proxy. The proxy further redirected to actual restcomm address (172.18.0.3).

In short the results of this testing endeavour were fair with no strong conclusions coming out of them.
